### PR TITLE
feature: default cookie banner

### PR DESCRIPTION
**Pull request by DonghyukJacobJang**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Code style update
- [ ] Refactor (refactoring or adding test which isn't a fix or add a feature)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [ ] No

**Did you test your solution?**

- [ ] I lightly tested it in one browser
- [ ] I deeply tested it in several browsers
- [ ] I wrote tests around it (unit tests, integration tests, E2E tests)

## Side Effects, Risks, Impact

- [ ] N/A

**Aditional comments:**

### CODE REVIEW

File: src/components/CookieBanner/CookieBanner.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+          <div className={styles.cookieSettingsContent}>
+            <p className={styles.cookieSettingsDescription}>
+              Ullamco deserunt dolore officia cillum ea culpa eu. Voluptate ex in commodo in dolor magna velit pariatur
+              in nostrud enim tempor aliquip nisi.
+            </p>
+
+            <ul>
+              <li>
+                <input type="checkbox" id="cookie-necessary" checked={cookieSettings.necessary} readOnly />
+                <label>necessary</label>
```
Comment (namniak) [inline src/components/CookieBanner/CookieBanner.js:62]: I believe all labels need to hafe `for` attribute with the id of the element they describe
with react it's be `htmlFor="cookie-necessary"` see https://reactjs.org/docs/accessibility.html#labeling

**Review by namniak — CHANGES_REQUESTED** [review]
just a minor note about labels

**Comment by DonghyukJacobJang** [comment]
@namniak Iran suggested to not add `noscript` tag at all because it is against the privacy law and very edge cases.

**Comment by namniak** [comment]
Agree - that's the best approach

**Pull request merged**